pygtfs/gtfs_entities.py

```diff
@@ -211,7 +216,31 @@ class Route(Base):
     trips = relationship("Trip", backref="route")
     fare_rules = relationship("FareRule", backref="route")
 
-    _validate_route_type = _validate_int_choice(range(8), 'route_type')
+    valid_route_types = range(8)
```

Comment:
no we do not. My first plan was to give the user the choice of allowing only the normal valid routes or the extended valid routes. But I didn't do that. In the same spirit, I didn't have to copy the test data set.